bluebrain/repo-bluebrain/packages/py-archngv/package.py

```diff
@@ -15,6 +15,8 @@ class PyArchngv(Package):
     version("develop", branch="main")
     version("2.0.0", tag="ArchNGV-v2.0.0")
 
+    variant("all", default=True, description="Circuit building support.")
```

Comment:
This isn't used by the package, though?

Reply:
Indeed. I removed it. I thought it was needed for the extras dependencies, but I already list them here so no need for this.